### PR TITLE
Fix the cenhubinsights.com redirects

### PR DESCRIPTION
**Pull request by rmulley**
TL;DR:
* I'm an idiot and misread the Jira story associated with this redirect. These changes fix the redirect introduced in #17.

## Story
[CenHub Insights DNS Update](https://uplightinc.atlassian.net/browse/GB-3532)

## Description
Uplight owns Central Hudson's insights domain https://www.cenhubinsights.com/. On 12/31/2020 Central Hudson would like Uplight to setup two redirects for them:

https://www.cenhubinsights.com/ -> https://www.cenhud.com/insights
https://cenhubinsights.com/ -> https://cenhud.com/insights

@gpnz has helped verify these domains in GCP. The next step is to merge this PR, and then on 12/31/2020 we can adjust DNS settings at https://www.name.com/account/domain/details/cenhubinsights.com#dns to use Google's nameserver, `ghs.googlehosted.com`, for both https://www.cenhubinsights.com and https://cenhubinsights.com

### CODE REVIEW

**Review by jacobrien — APPROVED** [review]

**Review by darren-hardy-uplight — APPROVED** [review]
Sorry I didn't catch that... Looks consistent with https://github.com/simpleenergy/se-engage-host-app/blob/master/server/config/utility-partners.config.js#L166

**Review by gpnz — APPROVED** [review]
thank you robert!
